app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -114,9 +114,10 @@ $include-html-global-classes: $include-html-classes;
 // $oil         : #333333;
 // $jet         : #222222;
 // $black       : #000000;
+   $maroon      : #5C1212;
```

Comment:
Color `#5C1212` should be written as `#5c1212`<br>Line should be indented 0 spaces, but was indented 3 spaces